projects/core/components/root/root.template.html

```diff
@@ -1,15 +1,17 @@
 <div class="t-root-content"><ng-content /></div>
-<tui-scroll-controls
-    *ngIf="scrollbars"
-    class="t-root-scrollbar"
-/>
-<ng-container #viewContainer />
-<ng-content select="tuiOverContent" />
-<tui-dialogs />
-<ng-content select="tuiOverDialogs" />
-<tui-alerts />
-<ng-content select="tuiOverAlerts" />
-<tui-dropdowns />
-<ng-content select="tuiOverDropdowns" />
-<tui-hints />
-<ng-content select="tuiOverHints" />
+<ng-container *ngIf="top()">
+    <tui-scroll-controls
+        *ngIf="scrollbars && !isChildRoot"
```

Comment:
Let's add `!isChildRoot` to `scrollbars`

Reply:
fixed